spec/actions_specs/reset_simulator_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe Fastlane do
+  describe Fastlane::FastFile do
+    it "raises an exception when no device name is specified" do
+      expect do
+        Fastlane::FastFile.new.parse("lane :test do
+          reset_simulator({})
+        end").runner.execute(:test)
+      end.to raise_error
```

Comment:
Thanks for the hint.

I've noticed that `FastlaneCore::Simulator.all` filters out any devices other than `iOS`.
Would it make sense to include Apple TVs and Apple Watches in the list too?
E.g. add `os_type` member to `Device` class and rename `ios_version` to `os_version`?
And maybe add helper methods like `ios_devices`, `apple_tvs` and `apple_watches` to return subset of `all`...

As a matter of fact, it seems to me now that the actual method to reset all or specific simulator should be added directly to `FastlaneCore::Simulator` and in Fastlane itself the action should be just a thin wrapper around that core method.

How does that sound?